addons/website_sale/models/product.py

```diff
@@ -384,6 +379,10 @@ def set_sequence_down(self):
         else:
             return self.set_sequence_bottom()
 
+    def set_sequence_reset(self):
+        rec = super(ProductTemplate, self).default_get(['website_sequence'])
```

Comment:
`self.default_get(['website_sequence'])` instead.

Unless I am missing something, `super(ProductTemplate, self)` and `self` are both `product.template` but by calling `default_get` on `super` we won't get self override of `default_get` in this file (there is no such override in this case but just in case).

It might also fit in one line if the super is removed.